plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Type.hs

```diff
@@ -165,3 +171,47 @@ erase (TPLC.Error ann _)            = Error ann
 -- | Erase a Typed Plutus Core Program to its untyped counterpart.
 eraseProgram :: TPLC.Program tyname name uni fun ann -> Program name uni fun ann
 eraseProgram (TPLC.Program a v t) = Program a v $ erase t
+
+{-|
+A specialized version of 'Term' for evaluation. The 'name' and 'a' parameters are specialized to 'Name' ('Unique') and '()' (omitted).
```

Comment:
Hasn't 'a' disappeared entirely?

Reply:
That's what I was getting at with "... specialized to () (omitted)". I can make it clearer.